test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -39,24 +39,53 @@ public ExportDataValidationTests(HttpIntegrationTestFixture fixture, ITestOutput
             _outputHelper = testOutputHelper;
         }
 
-        [Theory]
-        [InlineData("")]
-        [InlineData("Patient/")]
-        public async Task GivenFhirServer_WhenDataIsExported_ThenExportedDataIsSameAsDataInFhirServer(string path)
+        [Fact]
+        public async Task GivenFhirServer_WhenAllDataIsExported_ThenExportedDataIsSameAsDataInFhirServer()
         {
             // NOTE: Azure Storage Emulator is required to run these tests locally.
 
             // Trigger export request and check for export status
-            Uri contentLocation = await _testFhirClient.ExportAsync(path);
+            Uri contentLocation = await _testFhirClient.ExportAsync(string.Empty);
             IList<Uri> blobUris = await CheckExportStatus(contentLocation);
 
             // Download exported data from storage account
             Dictionary<(string resourceType, string resourceId), string> dataFromExport = await DownloadBlobAndParse(blobUris);
 
             // Download all resources from fhir server
-            Uri address = new Uri(_testFhirClient.HttpClient.BaseAddress, path);
+            Dictionary<(string resourceType, string resourceId), string> dataFromFhirServer = await GetResourcesFromFhirServer(_testFhirClient.HttpClient.BaseAddress);
+
+            // Assert both data are equal
+            Assert.True(ValidateDataFromBothSources(dataFromFhirServer, dataFromExport));
+        }
+
+        [Fact]
+        public async Task GivenFhirServer_WhenPatientDataIsExported_ThenExportedDataIsSameAsDataInFhirServer()
+        {
+            // NOTE: Azure Storage Emulator is required to run these tests locally.
+
+            // Trigger export request and check for export status
+            Uri contentLocation = await _testFhirClient.ExportAsync("Patient/");
+            IList<Uri> blobUris = await CheckExportStatus(contentLocation);
+
+            // Download exported data from storage account
+            Dictionary<(string resourceType, string resourceId), string> dataFromExport = await DownloadBlobAndParse(blobUris);
+
+            // Download all resources from fhir server
```

Comment:
nit: comment needs to be updated to `Patient` resources

Reply:
Fixed in the _type PR